crates/uv-python/python/get_interpreter_info.py

```diff
@@ -462,22 +462,30 @@ def get_operating_system_and_architecture():
         from .packaging._musllinux import _get_musl_version
 
         musl_version = _get_musl_version(sys.executable)
-        glibc_version = _get_glibc_version()
+
         if musl_version:
             operating_system = {
                 "name": "musllinux",
                 "major": musl_version[0],
                 "minor": musl_version[1],
             }
-        elif glibc_version != (-1, -1):
-            operating_system = {
-                "name": "manylinux",
-                "major": glibc_version[0],
-                "minor": glibc_version[1],
-            }
         else:
-            print(json.dumps({"result": "error", "kind": "libc_not_found"}))
-            sys.exit(0)
+            glibc_version = _get_glibc_version()
+
+            if glibc_version != (0, 0):
```

Comment:
I think this is indented too far, this should be an alternative to musl rather than in musl